### PR TITLE
Remade mod manager guide

### PR DESCRIPTION
**Pull request by Vladabdf**
It's now for the average user and changed the old guide into something to help mod authors. Includes an easy-to-follow numbered guide for curious mod authors who feel overwhelmed but want to try dipping their toes into KH modding.

Will go live [on the site](https://openkh.dev) soon after merge. Found at [/tool/GUI.ModsManager/](https://openkh.dev/tool/GUI.ModsManager/), which is currently hidden (but viewable with the URL) until I redo the entire organization of the site to be far, far more coherent and easy to follow than it currently is.

### CODE REVIEW

**Comment by Vladabdf** [comment]
This is ready to be merged

File: docs/tool/GUI.ModsManager/FAQ.md
Comment (shananas) [inline docs/tool/GUI.ModsManager/FAQ.md:32]: I'm not sure if it needs to be changed but Mod Manager specifically downloads Lua Backend for use with panacea. If the user does not install panacea Lua Backend will not load. IF the user manually installs Lua Backend without renaming it aka leaving it as `DBGHELP.dll` instead of renaming it to `LuaBackend.dll` mod manager will see it as not installed.
Basically as the main intention of mod manager is a mod loader using panacea the install + configuration of Lua Backend is specifically desgined for use with panacea. This can be changed if desired (it would require me to use some kind of hash to make sure its not another dll named `DBGHELP.dll`)so honestly probably not worth it

File: docs/tool/GUI.ModsManager/FAQ.md
Comment (shananas) [inline docs/tool/GUI.ModsManager/FAQ.md:26]: If a user wishes to use mod manager for patching and not for panacea/mod loader they can just not install panacea and the patching menu will be shown not the mod loader menu. this as mentioned in my prevous comment means they need to do any Lua Backend install and configuration manually. the intention of `dev view` is really for someone who made a mod and gets reports that its not working on patching to be able to test without uninstalling panacea (if they wish to troubleshoot the issue.) This is why i kinda hid it so idealy basic users wont see/try enabling it as patching then using panacea seems to cause many issues and anytime the kh2 rando community sees it said person is told to verify their game install remove the backup folder resetup mod manager etc. just too many variables to bother truely troubleshooting and just saying start nearly fresh. (only thing you dont need to do is uninstall and reinstall the game at least thats what i recall kh2 rando help doing)

File: docs/tool/GUI.ModsManager/FAQ.md
Comment (shananas) [inline docs/tool/GUI.ModsManager/FAQ.md:47]: This doesnt prevent it from opening it prevents mod manager from installing mods from github. Everything else works as expected

File: docs/tool/GUI.ModsManager/FAQ.md
Comment (shananas) [inline docs/tool/GUI.ModsManager/FAQ.md:24]: As I leave another i shoulve started a review but too late. to be completely truthful which im also not on the game extraction page. Any Lua only mod will work fine and any PCPatch will work fine. Basically any mod that just replaces an entire file outright will work for both panacea and patching. what doesnt work is mod manager functions beyond a simple copy. All binarcs, spawnpoint, listpatch, etc. Idk if you want to include any of this but the extract game data page for PC at least says Lua Scripts will work fine.

File: docs/tool/GUI.ModsManager/FAQ.md
Comment (shananas) [inline docs/tool/GUI.ModsManager/FAQ.md:28]: Mod Manager does make a backup of any pkg before patching but ofc if those arent vanilla for some reason then it wont matter. Also if a user goes into their game install folder after and messes with the backups mod manager wont know.

**Review by shananas — APPROVED** [review]
Now that im done not leaving all my comments as one review cause im dumb. most of that was more of an FYI overall i dont believe anything needs to be changed but will leave that up to you. Either merge it or if you wish make changes and merge. (if you do make changes ill be happy to take another look before the merge if wanted)

**Comment by Vladabdf** [comment]
I did indeed add some clarity to some of the points you mentioned. There were one or two that I thought were largely inconsequential or otherwise I covered any bases the majority of people would ask about. Since you went ahead and approved the PR earlier, with these changes, I'll go ahead and merge and we can make it official 👍

**Pull request merged**
